javacord-core/src/main/java/org/javacord/core/entity/server/ServerImpl.java

```diff
@@ -1405,7 +1407,22 @@ public void selfUndeafen() {
                 .execute(result -> {
                     List<Webhook> webhooks = new ArrayList<>();
                     for (JsonNode webhookJson : result.getJsonBody()) {
-                        webhooks.add(new WebhookImpl(getApi(), webhookJson));
+                        webhooks.add(WebhookImpl.createWebhook(getApi(), webhookJson));
+                    }
+                    return Collections.unmodifiableList(webhooks);
+                });
+    }
+
+    @Override
+    public CompletableFuture<List<IncomingWebhook>> getIncomingWebhooks() {
+        return new RestRequest<List<IncomingWebhook>>(getApi(), RestMethod.GET, RestEndpoint.SERVER_WEBHOOK)
+                .setUrlParameters(getIdAsString())
+                .execute(result -> {
+                    List<IncomingWebhook> webhooks = new ArrayList<>();
+                    for (JsonNode webhookJson : result.getJsonBody()) {
+                        if (webhookJson.get("type").asText().equals("1")) {
```

Comment:
Instead of having a [magic value](https://en.wikipedia.org/wiki/Magic_number_(programming)), it should use the `WebhookType` enum.
```suggestion
                        if (WebhookType.fromValue(webhookJson.get("type").asInt()) == WebhookType.INCOMING) {
```